config.yaml

```diff
@@ -0,0 +1,29 @@
+options:
+  email:
+    type: string
+    description: Account email address
+  server:
+    type: string
+    description: Certificate authority server
+    default: "https://acme-v02.api.letsencrypt.org/directory"
+  httpreq_endpoint:
+    type: string
+    description: URL of the HTTP endpoint to use
+  httpreq_http_timeout:
+    type: int
+    description: API request timeout
```

Comment:
might be nice to include the measurement s/m/h since this will be shown on charmcraft under `Reference`